docs/api/navigators/StackNavigator.md

```diff
@@ -95,6 +95,10 @@ Visual options:
 
 Generic title that can be used as a fallback for `headerTitle` and `tabBarLabel`
 
+#### `headerBackTitle`
```

Comment:
Note we don't have any of options listed there. My past commit just changed the list to references to this page. We might want to change that in the future if it's not the perfect UX https://reactnavigation.org/docs/navigators/navigation-options#Navigation-Option-Reference